### PR TITLE
Misc cleanups

### PR DESCRIPTION
**Pull request by gwynne**
- Updates min Swift version to 5.6 to match Vapor
- Updates CI (it's a Gwynne PR, what were you expecting?)
- Updates README

### CODE REVIEW

**Review by MahdiBM — APPROVED** [review]
I know this is going to surprise you, but LGTM!

File: README.md [hunk 1]
```diff
@@ -15,10 +15,13 @@
     <a href="LICENSE">
         <img src="http://img.shields.io/badge/license-MIT-brightgreen.svg" alt="MIT License">
     </a>
-    <a href="https://github.com/vapor/queues-redis-driver/actions">
-        <img src="https://github.com/vapor/queues-redis-driver/workflows/test/badge.svg" alt="Continuous Integration">
+    <a href="https://github.com/vapor/queues-redis-driver/actions/workflows/test.yml">
```
Comment (MahdiBM) [inline README.md:18]: Though, why is here 2 swift 5.6 and swift 5.8 badges? what about the poor 5.7?!

Comment (MahdiBM) [reply README.md:18]: I (obviously) like DiscordBM's approach which is dividing the versions by a " / " (I don't recall where i copied that from 😅):
```html
<a href="https://swift.org">
        <img src="https://img.shields.io/badge/swift-5.8%20/%205.7-brightgreen.svg" alt="Latest/Minimum Swift Version">
    </a>
```

Comment (gwynne) [reply README.md:18]: Hm... I definitely get the appeal of the "divider" approach. At the same time, I'm leaning towards changing my previous intent of showing both minimum and latest in favor of showing only the minimum, on the theory that the badge arguably suggests a claim that the package "has been updated for" the version on the badge, and that's certainly not the case for 5.8 in any package (with the exception of the `Exports.swift` bit), or even 5.7 in many of them. Thoughts?

Comment (MahdiBM) [reply README.md:18]: Yeah that seems reasonable :)
Generally the subject is subjective (😅) so I was just mentioning that there are alternative ways. But it still makes sense why they are there, like you said.

**Review by 0xTim — APPROVED** [review]

**Pull request merged**